Fix box shadow of header elements

MorrisJobke opened the pull request:
* unify shadow blur from 3px to 10px
* remove opacity of background of app labels
* for IE: use box-shadow as fallback (because the filter: drop-shadow is not supported)



Found while reviewing https://github.com/nextcloud/notifications/pull/89

cc @nextcloud/designers 

Before (opacity, ~~bigger blur, no shadow on notifications in Safari~~):

![bildschirmfoto 2017-12-19 um 12 10 16](https://user-images.githubusercontent.com/245432/34154499-fc075032-e4b5-11e7-963b-a0f8864a3a51.png)

After:

![bildschirmfoto 2017-12-19 um 12 09 56](https://user-images.githubusercontent.com/245432/34154525-11442574-e4b6-11e7-9a2a-ea92bdeecca1.png)

MorrisJobke committed:
Fix box shadow of header elements

* unify shadow blur from 3px to 10px
* remove opacity of background of app labels
* for IE: use box-shadow as fallback (because the filter: drop-shadow is not supported)

Signed-off-by: Morris Jobke <hey@morrisjobke.de>

jancborchardt commented:
Good stuff, but why reduce it from 10px which we used basically everywhere to 3px? It looks a bit strange and more like a thick border than a slight shadow.

MorrisJobke commented:
> Good stuff, but why reduce it from 10px which we used basically everywhere to 3px? It looks a bit strange and more like a thick border than a slight shadow.

Because we use that in our newer filters ;) See the line I just changed, so I just thought that we move to that. Otherwise we need to tough a lot more occurrences.

skjnldsv commented:
I disagree! The filter: drop-shadow is far better than the box-shadow as the arrow isn't included in the shadow generation. I'm against the removal of this. 

What's up with safari anyway? it should supports filter? https://caniuse.com/#feat=css-filters 😕

MorrisJobke commented:
Usually it does this, but somehow not for the notifications drop down. I will check tomorrow and adapt this pr then

MorrisJobke commented:
Okay - I moved to the 10px and to the `filter: drop-shadow`. I also added fixes for IE 11, because this supports only `box-shadow`. I tested it and it works now 👍
